frontend/package.json

```diff
@@ -3,22 +3,25 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
```

Comment:
When doing "sudo yarn install", I had the warnings :
warning "@emotion/styled > @emotion/babel-plugin@11.7.2" has unmet peer dependency "@babel/core@^7.0.0".
warning "@emotion/styled > @emotion/babel-plugin > @babel/plugin-syntax-jsx@7.16.5" has unmet peer dependency "@babel/core@^7.0.0-0".
warning "react-scripts > tailwindcss@3.0.8" has unmet peer dependency "autoprefixer@^10.0.2".
warning " > @typescript-eslint/eslint-plugin@5.8.1" has incorrect peer dependency "@typescript-eslint/parser@^5.0.0".
warning "react-scripts > eslint-config-react-app > eslint-plugin-flowtype@8.0.3" has unmet peer dependency "@babel/plugin-syntax-flow@^7.14.5".
warning "react-scripts > eslint-config-react-app > eslint-plugin-flowtype@8.0.3" has unmet peer dependency "@babel/plugin-transform-react-jsx@^7.14.9".

You can remove them if you want by changing the version of @typescript-eslint/parser to 5.8.1 and adding the lines:
    "@babel/core": "^7.11.0",
    "@babel/plugin-syntax-flow": "^7.16.5",
    "@babel/plugin-transform-react-jsx": "^7.16.5",
    "postcss": "^8.4.5",
    "autoprefixer": "^10.4.1",

Reply:
I have the same issue as @glerzing . However it is not only warnings for me, I can not launch the frontend.
The fix you are asking for is working for me.